test/systemtest/config.py

```diff
@@ -34,14 +35,14 @@
     },
     # The OTBN end-to-end tests can be run in simulation, but take a long time
     # there. Run them on FPGAs only for faster test results.
-    {
-        "name": "otbn_rsa_test",
-        "targets": ["fpga_nexysvideo"],
-    },
-    {
-        "name": "otbn_ecdsa_p256_test",
-        "targets": ["fpga_nexysvideo"],
-    },
+    #{
+    #    "name": "otbn_rsa_test",
+    #    "targets": ["fpga_nexysvideo"],
+    #},
+    #{
+    #    "name": "otbn_ecdsa_p256_test",
+    #    "targets": ["fpga_nexysvideo"],
+    #},
```

Comment:
I don't know how long these tests actually take in simulation. Since these are top-level end-to-end tests I think we should keep running them somewhere. Would it be feasible to run them in simulation until we enable OTBN on FPGA again? @imphil @rswarbrick @GregAC